config/config_test.go

```diff
@@ -37,6 +37,7 @@ func emptyConfig() *Config {
 		ResourceType{FilterRule{}, FilterRule{}},
 		ResourceType{FilterRule{}, FilterRule{}},
 		ResourceType{FilterRule{}, FilterRule{}},
+		ResourceType{FilterRule{}, FilterRule{}},
```

Comment:
If we aren't going to ensure that the config file can be used for API Gateway (with tests), is it worth adding it to the Config?

Reply:
Probably not! Added Config tests for both versions of API Gateway in [695ac8f](https://github.com/gruntwork-io/cloud-nuke/pull/352/commits/695ac8fd3835b72299fb1917e08c9e6f986663bc)

Reply:
Nicely done. 💯